src/components/TransactionPreview/ActionPreview.tsx

```diff
@@ -912,7 +924,7 @@ const ActionPreview = ({
         showCloseButton={showCloseButton}
         additionalTopButtons={additionalTopButtons}
       >
-        {isPolygonAccountWithEnoughPLR ? (
+        {!enableAssetSwap && !enableAssetBridge ? (
```

Comment:
you can put `!enableAssetSwap && !enableAssetBridge`  as constant as you using it multiple times
x